src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -87,8 +87,9 @@ def warnings
         if !boot_readable_by_grub?
           error_message =
             _(
-              "The boot loader cannot access the file system mounted at /boot. " \
-              "Only LUKS1 encryption is supported."
+              "The grub boot loader cannot access the file system mounted at /boot. " \
+              "Only LUKS1 and LUKS2 with PBKDF2 encryption is supported. An options would be to use "\
```

Comment:
I would reword this a bit: "Another boot loader can be selected in the boot loader section, for example  systemd_boot or grub2_bls."